python/chemiscope/__init__.py

```diff
@@ -6,6 +6,7 @@
     write_input,
 )
 from .jupyter import show, show_input  # noqa
+from .streamlit import viewer as streamlit_viewer  # noqa: F401
```

Comment:
why re-export under this name if the API example use `chemiscope.streamlit.viewer`?